dolphinscheduler-python/pydolphinscheduler/tests/resources_plugin/test_gitlab.py

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Test github resource plugin."""
+import pytest
+
+from pydolphinscheduler.resources_plugin.gitlab import GitLab
+
+
+@pytest.mark.parametrize(
+    "attr, expected",
+    [
+        (
+            "https://gitlab.com/chenruijie/ds-gitlab/-/blob/main/union.sh",
```

Comment:
can we remove all author-related message `chenruijie` to other like `pydolphinscheduler` or some thing else?
```suggestion
            "https://gitlab.com/pydolphinscheduler/ds-gitlab/-/blob/main/union.sh",
```

Reply:
> can we remove all author-related message `chenruijie` to other like `pydolphinscheduler` or some thing else?

i know